### DISCUSSION TITLE
Please report about jlmkr & Apps compatibility

### DISCUSSION THREAD

**Discussion by Jip-Hop** [Jip-Hop/jailmaker#26] [Polls]
Does using jailmaker allow you to use Apps side by side? Or does jailmaker cause issues or somehow interfere with the workings of TrueNAS SCALE Apps (Kubernetes)?

So far there are no reports of compatibility issues. Confirming that both work well side by side is helpful too (as [this has been requested by morganL from iXsystems](https://www.truenas.com/community/threads/best-way-to-run-vanilla-docker.108146/post-774600))!

Please vote and comment as this will help support [this pull request](https://github.com/truenas/scale-build/pull/482) to ensure systemd-nspawn stays packaged (again) with TrueNAS SCALE.

**Comment by spusuf** [comment]
I have disabled my apps (unset pool from settings in top right of Apps page) because even with NO APPS INSTALLED it was using AT LEAST 10% CPU in `top` and 5-20% on idle from the TrueNAS dashboard.
